airbyte-webapp/src/components/StreamTestingPanel/ConfigMenu.module.scss

```diff
@@ -0,0 +1,11 @@
+@use "scss/colors";
+
+.openModalButton.openModalButton {
+  height: 100%;
+}
+
+.modalContent {
+  width: 800px;
+  height: 700px;
```

Comment:
I ended up just using the Modal component's `size` prop in order to set the width, and that seems to react to narrow screen widths as desired (it shrinks so that it is fully contained by the screen even for a narrow viewport). Updated the height to `60vh` since that felt right